app/src/main/java/com/internship/move/utils/InternalStorageManager.kt

```diff
@@ -0,0 +1,28 @@
+package com.internship.move.utils
+
+import android.content.Context
+import android.content.Context.MODE_PRIVATE
+import com.internship.move.utils.Constants.SharedPref.KEY_APP_PREFERENCES
+import com.internship.move.utils.Constants.SharedPref.KEY_HAS_VISITED_AUTHENTICATION
+import com.internship.move.utils.Constants.SharedPref.KEY_HAS_VISITED_ONBOARDING
+
+class InternalStorageManager(context: Context) {
+
+    private val sharedPref = context.getSharedPreferences(KEY_APP_PREFERENCES, MODE_PRIVATE)
```

Comment:
Personally I prefer storing the KEY constants in the `InternalStorageManager` companion object. Imo it is easier to see them where they are used.
Of course, this approach might be problematic when the `InternalStorageManager` expands so much that the `companion object` becomes huge too 😬